cookbooks/upstage/financial_analysis.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        print(search.page_content)
</pre><p>It may be more useful to print the <code>doc_base</code> rather than the html. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UpstageAI/cookbook/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>